thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertJobUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.linkedin.thirdeye.anomaly.alert;
+
+import org.apache.commons.mail.DefaultAuthenticator;
+import org.apache.commons.mail.EmailException;
+import org.apache.commons.mail.HtmlEmail;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linkedin.thirdeye.anomaly.ThirdeyeSmtpConfiguration;
+
+public class AlertJobUtils {
+  public static final String EMAIL_ADDRESS_SEPARATOR = ",";
+  private static final Logger LOG = LoggerFactory.getLogger(AlertJobUtils.class);
+
+  public static void sendEmailWithTextBody(ThirdeyeSmtpConfiguration smtpConfigutation, String subject,
+      String textBody) throws EmailException {
+    HtmlEmail email = new HtmlEmail();
+    email.setSubject(subject);
+    email.setTextMsg(textBody);
+    sendEmail(smtpConfigutation, email);
+  }
+
+  public static void sendEmailWithHtml(ThirdeyeSmtpConfiguration smtpConfiguration, String subject,
+      String htmlBody) throws EmailException {
+    HtmlEmail email = new HtmlEmail();
+    email.setSubject(subject);
+    email.setHtmlMsg(htmlBody);
+    sendEmail(smtpConfiguration, email);
+  }
+
+  /** Sends email according to the provided config. This method does not support html emails. */
```

Comment:
Judging from the usage of this method in sendEmailWithHtml, I'm guessing this comment is no longer relevant. Feel free to remove it.